### DISCUSSION TITLE
Add someway for osu main event (OWC) being able to get imported into calendar

### DISCUSSION THREAD

**Discussion by mixalisnew** [ppy/osu#30581] [Ideas] [answered]
I dont know if this is the right place for this but here goes nothing. I have this problem not being able tofollow anything that is not in my local time zone and also i like having those things in my calendar, so i was wandering if something could be made for the OWC schedule to update automaticly in someones calendar maybe threw an API? i dont know if i use the term for this right i got the idea from  a guy who does the same thing for League of Legends Esposrts https://github.com/ArnoutPullen/LolEsportsCalendar.

if this discustion is out of place please let me know and i will delete or close it. Thanks for your time.

**Comment by mixalisnew** [comment]
since lazer grand arena got announced today bumping this up to get people's opinion again. for me that would be huge

**Comment by peppy** [comment, marked as answer]
Would the easiest way not be to have a shared google calendar with all events? Or .ics files I guess..

**Reply by mixalisnew** [reply]
if that works for sure

**Reply by mixalisnew** [reply]
are we doing anything for this years owc?
